src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test.Common/LanguageServerTestBase.cs

```diff
@@ -46,6 +46,8 @@ public SingleThreadedForegroundDispatcher()
             private Thread Thread { get; } = Thread.CurrentThread;
 
             public override bool IsForegroundThread => Thread.CurrentThread == Thread;
+
```

Comment:
Anything below this point can be ignored, it's all tests and I don't have them set up to work properly currently